appeals/web/src/server/appeals/appeal-details/service-user/service-user.mapper.js

```diff
@@ -63,6 +72,20 @@ export const changeServiceUserPage = (appealData, userDetailsInSession, userType
 					errorMessage: errorLastName(errors)
 				}
 			},
+			{
+				type: 'input',
+				parameters: {
+					id: 'org-name',
```

Comment:
Very minor and pedantic but this might be better un-abbreviated as `organisation-name` and `organisationName` below